homeassistant/components/light/__init__.py

```diff
@@ -107,6 +110,7 @@
     ATTR_WHITE_VALUE: vol.All(vol.Coerce(int), vol.Range(min=0, max=255)),
     ATTR_FLASH: vol.In([FLASH_SHORT, FLASH_LONG]),
     ATTR_EFFECT: cv.string,
+    vol.Optional(ATTR_POWER_ON, default=True): cv.boolean,
```

Comment:
this just doesn't make much sense… turn on with power off? 😛  I wonder if we should extend the turn_off schema instead?